docs/Building-Transmission.md

```diff
@@ -5,6 +5,8 @@ The source code for both official and nightly releases can be found on our [down
 Software prerequisites:
  * macOS 10.14.4 or newer
  * Xcode 11.3.1 or newer
+ * Node.js 10 or newer
+ * Yarn
```

Comment:
Do we still need Yarn after #4637 ?

Reply:
No. It's 2am here so I'm not pushing any more commits tonight, but I'll leave this convo unresolved to remind me when I come back to this